spec/outputs/s3_spec.rb

```diff
@@ -211,7 +211,7 @@
     context 'us-east-1' do
       let(:region) { 'us-east-1' }
       it "sets endpoint" do
-        expect( subject.send(:bucket_resource).client.config.endpoint.to_s ).to eql 'https://s3.us-east-1.amazonaws.com'
+        expect( subject.send(:bucket_resource).client.config.endpoint.to_s ).to eql 'https://s3.amazonaws.com'
```

Comment:
The default region of s3 is us-east-1. The endpoint returns from SDK has changed